### DISCUSSION TITLE
Add advertising (preroll)

### DISCUSSION THREAD

**Discussion by greenRittu** [mediacms-io/mediacms#752] [Feature Requests]
Hi!

Could you integrate advertising (as a preroll).
Or help me with it. I can do it with videojs player + videojs.ads.js, videojs-preroll.js and it works!

```
  <script src="lib/videojs.ads.js"></script>
  <script src="src/videojs-preroll.js"></script>

  <script>
    // fire up the plugin
    videojs('examplePlayer', {}, function(){
      var player = this;
      player.preroll({
        src: {src: "https://vjs.zencdn.net/v/oceans.mp4", type: "video/mp4"},
        href: "http://videojs.com",
        adsOptions: {debug: false}
      });
    });
  </script>
```

![SCR](https://user-images.githubusercontent.com/10289241/229752108-19cfbe3a-f61c-4326-8b4c-58abd7b0d9b1.JPG)

But how can I use it with mediacms player? It blocks not all GUI :c

**Comment by streamly** [comment]
I would like to see how this can be implemented too. You have the plugins in two different folders lib/ and /src ... is this an error?

**Reply by greenRittu** [reply]
So can it be developed? Or can someone help? :c
